src/include/utils.h

```diff
@@ -1341,6 +1341,15 @@ pgmoneta_backtrace(void);
 int
 pgmoneta_os_kernel_version(char** os, int* kernel_major, int* kernel_minor, int* kernel_patch);
 
+/**
+ * Write info to a file
+ * @param sfile The file to write to
+ * @param fmt The format string
+ * @param ... The arguments to be formatted
+ */
+void
+pgmoneta_write_info(FILE* sfile, const char* fmt, ...);
```

Comment:
Can this just be a static function inside info.c?

Reply:
@Arshdeep54 I would expect that we would end up with a `pgmoneta_load_info` and a `pgmoneta_save_info` in `info.h` and the rest be static inside `info.c`